Description: Fix case of "OpenSSL" to match the case of the file shipped by cmake
Author: Mattia Rizzolo <mattia@debian.org>
Last-Update: 2017-06-05

--- a/cmake/modules/FindLIBPODOFO.cmake
+++ b/cmake/modules/FindLIBPODOFO.cmake
@@ -45,7 +45,7 @@
 	#message(STATUS "PODOFO PATCH VERSION ${PODOFO_PATCH_VER}")
 	set(LIBPODOFO_VERSION "${PODOFO_MAJOR_VER}.${PODOFO_MINOR_VER}.${PODOFO_PATCH_VER}" CACHE STRING "PoDoFo version string")
 	if(LIBPODOFO_VERSION VERSION_GREATER "0.9.4")
-		find_package(OPENSSL)
+		find_package(OpenSSL)
 		if (OPENSSL_FOUND)
 			message("OpenSSL found OK for installed version of PoDoFo (>= 0.9.5) - Enabling support for PDF embedded in AI")
 			set(LIBPODOFO_FOUND TRUE CACHE BOOLEAN "Was libpodofo found")
